lapce-ui/src/app.rs

```diff
@@ -247,159 +247,196 @@ fn macos_window_desc<T: druid::Data>(desc: WindowDesc<T>) -> WindowDesc<T> {
-                            data: None,
-                        },
-                        Target::Auto,
-                    )))
-                    .entry(MenuItem::new("Close window").command(Command::new(
-                        LAPCE_COMMAND,
-                        LapceCommand {
-                            kind: CommandKind::Workbench(
-                                LapceWorkbenchCommand::CloseWindow,
-                            ),
-                            data: None,
-                        },
-                        Target::Auto,
-                    ))),
-            )
-            .entry(
-                Menu::new("Edit")
-                    .entry(MenuItem::new("Cut").command(Command::new(
-                        LAPCE_COMMAND,
-                        LapceCommand {
-                            kind: CommandKind::Edit(EditCommand::ClipboardCut),
-                            data: None,
-                        },
-                        Target::Auto,
-                    )))
-                    .entry(MenuItem::new("Copy").command(Command::new(
-                        LAPCE_COMMAND,
-                        LapceCommand {
-                            kind: CommandKind::Edit(EditCommand::ClipboardCopy),
-                            data: None,
-                        },
-                        Target::Auto,
-                    )))
-                    .entry(MenuItem::new("Paste").command(Command::new(
-                        LAPCE_COMMAND,
-                        LapceCommand {
-                            kind: CommandKind::Edit(EditCommand::ClipboardPaste),
-                            data: None,
-                        },
-                        Target::Auto,
-                    )))
-                    .separator()
-                    .entry(MenuItem::new("Undo").command(Command::new(
-                        LAPCE_COMMAND,
-                        LapceCommand {
-                            kind: CommandKind::Edit(EditCommand::Undo),
-                            data: None,
-                        },
-                        Target::Auto,
-                    )))
-                    .entry(MenuItem::new("Redo").command(Command::new(
-                        LAPCE_COMMAND,
-                        LapceCommand {
-                            kind: CommandKind::Edit(EditCommand::Redo),
-                            data: None,
-                        },
-                        Target::Auto,
-                    )))
-                    .separator()
-                    .entry(MenuItem::new("Find").command(Command::new(
-                        LAPCE_COMMAND,
-                        LapceCommand {
-                            kind: CommandKind::Focus(FocusCommand::Search),
-                            data: None,
-                        },
-                        Target::Auto,
-                    ))),
-            )
+                            .hotkey(SysMods::Cmd, "q")
```

Comment:
ditto

Reply:
I forgot that macOS platform has this binding no matter what we do, so I guess this one can stay?